pkg/base1/cockpit.js

```diff
@@ -1853,6 +1854,29 @@ function full_scope(cockpit, $, po) {
             return new DBusProxies(self, cache, String(iface), String(path_namespace));
         };
 
+        self.poll = function poll(path, iface) {
```

Comment:
True, it could be an optional setting. Perhaps an argument to the "watch" message? Because we already 'poke' whenever there is an outgoing message.